src/registry/content/Content.scss

```diff
@@ -0,0 +1,76 @@
+@use '@/scss/bem';
+@use '@utrecht/components/link';
+@use 'microscope-sass/lib/typography';
```

Comment:
`microscope-sass` is on the blocklist - it is not to be used in new code anymore (and the formio-renderer is new code). We consider it technical debt, because it relies on global variables which conflicts with the sass module system (`@use` vs. `@import`) and it hardcodes the font color/family etc. rather than allowing us to use design tokens.